### PR TITLE
Show poi name of the post in feed and myPosts

### PR DESCRIPTION
**Pull request by dyxcooh**
Display the name of POI for each post when it is viewed in my posts or feed. 
Align 'sort by' text with its spinner

### CODE REVIEW

File: app/src/main/java/com/github/epfl/meili/posts/PostListRecyclerAdapter.kt [hunk 1]
```diff
@@ -37,6 +38,7 @@ class PostListRecyclerAdapter(private val viewModel: PostListViewModel, private
             MeiliWithUserRecyclerViewHolder<Post>(itemView, listener) {
         private val title: TextView = itemView.findViewById(R.id.post_title)
         private val postId: TextView = itemView.findViewById(R.id.post_id)
+        private val poiName: TextView = itemView.findViewById(R.id.ppoi_name)
```
Comment (aman3014) [inline app/src/main/java/com/github/epfl/meili/posts/PostListRecyclerAdapter.kt:41]: typo

File: app/src/main/java/com/github/epfl/meili/posts/PostListRecyclerAdapter.kt [hunk 2]
```diff
@@ -48,8 +50,11 @@ class PostListRecyclerAdapter(private val viewModel: PostListViewModel, private
             super.bind(user, post)
 
             postId.text = post.postId()
+            poiName.text = post.poiName
             title.text = post.title
 
+            if(showPOI){poiName.visibility=View.VISIBLE}
```
Comment (aman3014) [inline app/src/main/java/com/github/epfl/meili/posts/PostListRecyclerAdapter.kt:56]: autoformatting

**Review by aman3014 — CHANGES_REQUESTED** [review]
Thanks for this feature! I just left a couple of minor remarks.

Also, do you think you could add very simple tests in Feed, Forum, and MyPosts to assert that the poi name is shown or not shown?

**Comment by dyxcooh** [comment]
> Thanks for this feature! I just left a couple of minor remarks.
> 
> Also, do you think you could add very simple tests in Feed, Forum, and MyPosts to assert that the poi name is shown or not shown?

Since the posts are showed as postList in these activities and identified by a single id per list, I cannot access the poi name using an id and do check(matches(isDisplayed())). Do you have an idea about how to test them in mind?

**Comment by aman3014** [comment]
> Since the posts are showed as postList in these activities and identified by a single id per list, I cannot access the poi name using an id and do check(matches(isDisplayed())). Do you have an idea about how to test them in mind?

I think you can use withText instead of withId. Just be careful because it doesn't work if there are multiple matching views. So you'll need to either give different names to pois or make a test with a single poi in the list.

**Review by aman3014 — APPROVED** [review]

**Review by tberkane — APPROVED** [review]

**Pull request merged**